packages/nimble-components/src/anchor-tabs/specs/README.md

```diff
@@ -0,0 +1,206 @@
+
+---
+
+## Implementation
+
+As long as there is at least one non-disabled tab in the tabs group, we will have an active tab. Upon connecting `nimble-navigation-tabs` to the DOM, we will activate (i.e. navigate to) the tab specified as initially active. If no tab was specified as initially active, we will default to the first non-disabled tab.
+
+Each `nimble-navigation-tab` must have an id so that `nimble-navigation-tabs`'s `activeid` has a valid value. If the user does not specify an id for a tab, we will generate one using FAST's `uniqueId()` utility function.
+
+If the active tab becomes disabled, it will remain active. We will not automatically activate a different tab.
+
+When the tabs group gains or loses focus we must set or remove the `focused` class from one of the child tabs. To do this, we will add `onfocus` and `onblur` event handlers to the tabs group.
+
+
+### States
+
+#### Keyboard focused state
+When tabbing to the `nimble-navigation-tabs`, it gets keyboard focus. To indicate this, the active tab (which is already underlined) gets a second, thinner underline.
+
+### Accessibility
+
+- *Keyboard Navigation and Focus*
+    -   The tabs control will have a visual indication of which tab is active (green underline).
+    -   Once focused, the tabs control will show a visual indication of which tab has focus, starting with the active tab.
+    -   Arrowing left or right will change the focused tab. Arrowing left on the first tab will wrap around to the last, and vice versa.
+    -   `Home` will move focus to the first (non-disabled) tab.
+    -   `End` will move focus to the last (non-disabled) tab.
+    -   The `Enter` or `Space` key will activate the focused tab, moving the active indicator and loading the tab's associated URL.
+- *Form Input*
+    - N/A
+- *Use with Assistive Technology*
+    - We cannot use the `tablist`, `tab`, and `tabpanel` roles because we have no `tabpanel`s. Instead we use a `nav` container element (which has role `navigation`) and `link` roles on each of our tabs. An element with role `navigation` should have a label that describes the navigation group. The user will have to provide this, so we will propagate the `aria-label` attribute value to the `nav` element in the shadow DOM.
+
+### Globalization
+
+N/A
+
+### Security
+
+N/A
+
+### Performance
+
+N/A
+
+### Dependencies
+
+None
+
+### Test Plan
+
+Unit tests will be created that exercise the API.
+
+### Tooling
+
+N/A
+
+### Documentation
+
+We will create a new Storybook entry for navigation tabs that documents the API.
```

Comment:
It would be good to also include a sentence of usage guidance in both the original tab and the new anchor tab. Hopefully they can point to each other and explain when it's appropriate to use each.

Looks like you already did this for button -> anchor-button 👍

Reply:
Added a note about this.